lib/internal/Magento/Framework/composer.json

```diff
@@ -24,7 +24,7 @@
         "lib-libxml": "*",
         "colinmollenhour/php-redis-session-abstract": "~1.3.8",
         "composer/composer": "^1.6",
-        "magento/zendframework1": "~1.13.0",
+        "magento/zendframework1": "dev-master",
```

Comment:
Will this work as expected without specifying the zf1 git repo, as in the base composer.json

Reply:
Good catch. I added repositories.

Reply:
@danielrenaud , You can only specify repositories in the root package.  If you want to learn why, they have a FAQ on it https://getcomposer.org/doc/faqs/why-can't-composer-load-repositories-recursively.md